tests/testthat/test-rd-html.R

```diff
@@ -159,6 +159,10 @@ test_that("can control \\Sexpr output", {
   expect_equal(rd2html("\\Sexpr[results=hide]{1}"), character())
   expect_equal(rd2html("\\Sexpr[results=text]{1}"), "1")
   expect_equal(rd2html("\\Sexpr[results=rd]{\"\\\\\\emph{x}\"}"), "<em>x</em>")
+  expect_equal(rd2html("\\Sexpr[results=verbatim]{1 + 2}"),
+               c("<pre>", "[1] 3", "</pre>"))
+  expect_equal(rd2html("\\Sexpr[results=verbatim]{cat(42)}"),
```

Comment:
Can you please include a test for mingling implicitly output and side effects?

And can you please indent following https://style.tidyverse.org/syntax.html#long-lines?

Reply:
Assuming you meant something like `{cat('42!\n'); 3}`, I added that in the amended commit.

I forgot to say: Please feel free to amend this patch/branch as needed.